src/doc/trpl/guessing-game.md

```diff
@@ -239,7 +239,7 @@ use std::rand;
 fn main() {
     println!("Guess the number!");
 
-    let secret_number = (rand::random::<uint>() % 100u) + 1u;
+    let secret_number = (rand::random::<u32>() % 100u) + 1u;
```

Comment:
This is currently breaking the build:
```
<anon>:10:50: 10:54 error: mismatched types: expected `u32`, found `usize` (expected u32, found usize)
<anon>:10     let secret_number = (rand::random::<u32>() % 100u) + 1u;
                                                           ^~~~
<anon>:10:58: 10:60 error: mismatched types: expected `u32`, found `usize` (expected u32, found usize)
<anon>:10     let secret_number = (rand::random::<u32>() % 100u) + 1u;
                                                                   ^~
```
because the `u` suffix means that those literals are of type `uint` still.  As it turns out, these suffixes can actually be entirely removed now due to integer fallback, e.g. this can become:
```rust
let secret_number = (rand::random::<u32>() % 100) + 1;
```
both here, and at the several other occurrences of this same line below here, and that will fix it. :)